mesa/space.py

```diff
@@ -435,24 +435,6 @@ def move_agent(self, agent: Agent, pos: Coordinate) -> None:
         self.remove_agent(agent)
         self.place_agent(agent, pos)
 
-    def place_agent(self, agent: Agent, pos: Coordinate) -> None:
```

Comment:
I think it should be an abstract method that raises `NotImplementedError` here.

Reply:
Actually, should be just `...`. It's an abstract method after all, and to be consistent with other methods.

Reply:
ok, used the `...` syntax